python/paddle/fluid/incubate/fleet/collective/__init__.py

```diff
@@ -117,23 +78,65 @@ def save_inference_model(self,
                              main_program=None,
                              export_for_deployment=True):
         io.save_inference_model(dirname, feeded_var_names, target_vars,
-                                self._executor, main_program, None, None,
+                                executor, main_program, None, None,
                                 export_for_deployment)
 
     def save_persistables(self, executor, dirname, main_program=None):
-        io.save_persistables(self._executor, dirname, main_program, None)
+        io.save_persistables(executor, dirname, main_program, None)
+
+    def node_num(self):
```

Comment:
I think this function is not in fleet_base.py, we have a protocol that all sub class implementation should follow fleet_base.py inferface design.

Reply:
Done.

Reply:
Done.